pkgs/tools/video/yamdi/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  fetchurl,
+  ...
```

Comment:
Didn't realize that. Thanks!

On Wed, Mar 16, 2016 at 11:02 AM, Christoph Hrdinka <
notifications@github.com> wrote:

> In pkgs/tools/video/yamdi/default.nix
> <https://github.com/NixOS/nixpkgs/pull/13962#discussion_r56349403>:
>
> > @@ -0,0 +1,36 @@
> > +{
> > +  stdenv,
> > +  fetchurl,
> > +  ...
>
> You can omit the dots here. These are used to match all remaining
> attributes in an attribute set. This is needed if your package is called
> with a fixed set of arguments or when you don't know all inputs/only want
> to pick needed ones. callPackage however will only pass needed packages.
> You can further put this in one line.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/13962/files/40a1d6e14554c0b13631738b3c696f00a8f2123d#r56349403>
>



-- 
Ryan Artecona